src/config.rs

```diff
@@ -1126,6 +1126,25 @@ impl DbConfig {
                 return Err("pipelined_write is not compatible with unordered_write".into());
             }
         }
+
+        // Since the following configuration supports online update, in order to
+        // prevent mistakenly inputting too large values, the max limit is made
+        // according to the cpu quota.
+        let cpu_num = SysQuota::cpu_cores_quota() as i32;
+        if self.max_background_jobs <= 0 || self.max_background_jobs > cpu_num {
```

Comment:
I'm not sure if this limitation could fit in these config items. cc @Little-Wallace @Connor1996

Reply:
lgtm